src/main/java/com/next/step/step2/StringSplit.java

```diff
@@ -0,0 +1,24 @@
+package com.next.step.step2;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringSplit {
```

Comment:
```suggestion
public class SplitedString {
```
- 클래스 이름은 명사형으로 작성하는건 어떨까요?
- 객체를 만들라는 의미는 유틸성 클래스가 아니었어요. 객체의 생성자에서 유효성검증을 해서 이 객체를 사용하는 측에서는 이를 보장해주면 되지 않을까했어요. 유틸성 클래스로 만들다보니 er 등 네이밍에 대한 고민이 생겼었군요. 저도 엘레강트 오브젝트의 해당 부분에 대해서는 동의하는 편입니다.